src/Data/Argonaut/Extension/Func.purs

```diff
@@ -0,0 +1,24 @@
+module Data.Argonaut.Extension.Func where
+
+import Data.Argonaut.Encode
+import Data.Argonaut.Core
+import Data.Function
+
+foreign import encodeFuncs """
```

Comment:
Functions are not a valid `Json` type, so I don't think this should exist. For places in the API where functions or values are allowed, we should use `Either` or something similar. These instances can break a lot of code.